subscription_management/lambda_function.py

```diff
@@ -101,14 +108,14 @@ def lambda_handler(event, context):
         # A future improvement could be made to authenticate this, but for the moment,
         # I don't forsee anyone DoS-ing this endpoint causing any harm.
         sms_file_id, phone_file_id = upload_telephony_contact_list(body.get('category'))
-        return respond_success("{\"status\": \"success\", \"sms_file_id\": \"%s\", \"phone_file_id\": \"%s\"}" % (sms_file_id, phone_file_id), origin)
+        return respond_success(f"{{\"status\": \"success\", \"sms_file_id\": \"{sms_file_id}\", \"phone_file_id\": \"{phone_file_id}\"}}", origin)
     elif method == 'POST' and path == 'verify':
         # Fire off email to confirm the user is allowed to make subscription changes.
         locally_computed_hmac = hmac.new(hmacsecret, codecs.encode(email), hashlib.sha256).digest()
         signature = base64.urlsafe_b64encode(locally_computed_hmac).decode('utf8').rstrip("=")
-        msg_body = "Please visit the following URL to verify your email address: " \
-                   "https://bugalert.org/content/pages/my-subscriptions.html" \
-                   "?signature={}&email={}".format(signature, email)
+        msg_body = f"Please visit the following URL to verify your email address: "
+                   f"https://bugalert.org/content/pages/my-subscriptions.html"
+                   f"?signature={signature}&email={email}"
```

Comment:
```suggestion
        msg_body = f"Please visit the following URL to verify your email address: " \
                   f"https://bugalert.org/content/pages/my-subscriptions.html" \
                   f"?signature={signature}&email={email}"
```

Multiline requires the EOL backslashes.